site/content/en/docs/Reference/API/replacements.md

```diff
@@ -0,0 +1,387 @@
+---
+title: "replacements"
+linkTitle: "replacements"
+type: docs
+weight: 18
+description: >
+    Substitute field(s) in N target(s) with a field from a source.
+---
+
+Replacements are used to copy fields from one source into any
+number of specified targets.
+
+\
+The `replacements` field can support a path to a replacement:
+
+`kustomization.yaml`
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+replacements:
+  - path: replacement.yaml
+```
+`replacement.yaml`
+```yaml
+source:
+  kind: Deployment
+  fieldPath: metadata.name
+targets:
+  - select:
+      name: my-resource
+```
+\
+Alternatively, `replacements` supports inline replacements:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+replacements:
+- source:
+    kind: Deployment
+    fieldPath: metadata.name
+  targets:
+  - select:
+      name: my-resource
+```
+
+### Syntax
+
+The full schema of `replacements` is as follows:
+
+```yaml
+replacements:
+- source:
+    group: string
+    version: string
+    kind: string
+    name: string
+    namespace: string
+    fieldPath: string
+    options:
+      delimiter: string
+      index: int
+      create: bool
+  targets:
+  - select:
+      group: string
+      version: string
+      kind: string
+      name: string
+      namespace: string
+    reject:
+    - group: string
+      version: string
+      kind: string
+      name: string
+      namespace: string
+    fieldPaths:
+    - string
+    options:
+      delimiter: string
+      index: int
+      create: bool
+```
+
+### Field Descriptions
+
+| Field       | Required| Description | Default |
+| -----------: | :----:| ----------- | ---- |
+| `source`| :heavy_check_mark: | The source of the value |
```

Comment:
The `:heavy_check_mark:` doesn't appear to render. I am not sure if this is an issue with how Hugo is configured. [This](https://discourse.gohugo.io/t/emojis-not-rendering-with-enableemoji-true/41011) suggests we might be able to simply paste the emoji. Alternatively, we could use a different character or format to indicate that a field is required.

Reply:
Maybe we should apply changes in this PR. This one's not approved but it fixes the emoji issue
- https://github.com/kubernetes-sigs/cli-experimental/pull/355

Reply:
Fixed it. https://github.com/kubernetes-sigs/kustomize/pull/5313/commits/8788346e5c4226e21d73be17bc948eabb5d6c622

Reply:
https://github.com/kubernetes-sigs/kustomize/pull/5313/commits/bd8045b64874eb8aaccfe4aef639a0d375b6e428

![image](https://github.com/kubernetes-sigs/kustomize/assets/73531614/ad81dbef-aed8-4c0a-bea9-5cfe44d4a4cb)

Reply:
Nice catch! I looked into this issue briefly. I think the current approach of using the character works.

It seems that the original author tried to use Github syntax for emojis. We should be able to render characters like check marks also in html with Docsy [shortcodes](https://www.docsy.dev/docs/adding-content/shortcodes/) or Docsy's [integration](https://www.docsy.dev/docs/adding-content/shortcodes/) with Latex.